CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+### 2.29.0
+This release includes:
+* Fluent Bit [1.9.10](https://fluentbit.io/announcements/v1.9.10/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.1
+* Amazon Kinesis Streams for Fluent Bit 1.10.1
+* Amazon Kinesis Firehose for Fluent Bit 1.7.1
+
+Compared to `2.28.4` this release adds:
+* Feature - Add `store_dir_limit_size` option [fluent-bit-docs:971](https://github.com/fluent/fluent-bit-docs/pull/971)
+* Feature - New filter for AWS ECS Metadata [fluentbit:5898](https://github.com/fluent/fluent-bit/pull/5898)
+* Enhancement - Resolve Fluent Bit networking hangs affecting CloudWatch plugin by migrating to async networking + sync core scheduler [fluent-bit:6339](https://github.com/fluent/fluent-bit/pull/6339)
+* Enhancement - Different user agent on windows vs linux [fluentbit:6325](https://github.com/fluent/fluent-bit/pull/6325)
+
+Compared to `2.28.4` this release adds the following feature that we are working on getting accepted upstream:
+
+* Feature - Kinesis FireHose and Streams nanoseconds and milliseconds `time_key_format` support [fluentbit:2831](https://github.com/fluent/fluent-bit/issues/2831)
+* Bug - Use log timestamp as s3 file timestamp [aws-for-fluent-bit:460](https://github.com/aws/aws-for-fluent-bit/issues/460)
```

Comment:
nit: `Use log timestamp for s3 key timestamp` is more clear IMO

Reply:
Great. Thanks